backend/backend/settings.py

```diff
@@ -157,10 +159,11 @@
     },
     'mongodb': {
         'ENGINE': 'djongo',
-        'NAME': os.getenv('MONGO_DB_NAME'),  # MongoDB database name
+        'NAME': os.getenv('MONGO_DB_NAME'),
         'ENFORCE_SCHEMA': False,
         'CLIENT': {
-            'host': os.getenv('MONGO_URI'),  # MongoDB URI
+            'host': os.getenv('MONGO_URI'),
+            'tlsCAFile': certifi.where(),  # Ensure SSL verification with certifi
```

Comment:
What's the purpose of this?

Reply:
sometimes database gives error of SSL handshake, for this we have to add this in connection. certifi.where() points to the CA certificates used by python's requests library to verify the identity of the server we are connecting to, also  it ensures that out application is connecting to a legit mongodb server and not a fake one.

https://stackoverflow.com/questions/54484890/ssl-handshake-issue-with-pymongo-on-python3